src/plugins/saved_objects_management/public/management_section/objects_table/components/flyout.tsx

```diff
@@ -78,8 +79,10 @@ import { FailedImportConflict, RetryDecision } from '../../../lib/resolve_import
 import { OverwriteModal } from './overwrite_modal';
 import { ImportModeControl, ImportMode } from './import_mode_control';
 import { ImportSummary } from './import_summary';
+// eslint-disable-next-line @osd/eslint/no-restricted-paths
```

Comment:
Do we need this?

Reply:
Yes, this is required to avoid lint error

Reply:
I think if we take care of [below comment](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5781#discussion_r1476707363), then we won't have lint errors.

Reply:
Moved export to top level and this line is removed in new commit